cmds/update.js

```diff
@@ -0,0 +1,321 @@
+ */
+const updateJobSnapshot = async (option) => {
+  const ezunpaywall = await connection();
+  const config = await getConfig();
+
+  const data = {};
+
+  if (option.index) data.index = option.index;
+
+  try {
+    await ezunpaywall({
+      method: 'POST',
+      url: '/api/update/job/period',
+      data,
+      headers: {
+        'x-api-key': config.apikey,
+      },
+    });
+  } catch (err) {
+    logger.errorRequest(err);
+    process.exit(1);
+  }
+  logger.info('Insert current snapshot');
+
+  if (option.verbose) {
+    await verbose();
+  }
+  process.exit(0);
+};
+
+/**
+ * get report of update process
+ *
+ * @param {String} option.file --file <file> | report filename on ezunpaywall
+ * @param {Boolean} option.latest --latest | latest report on ezunpaywall
+ * @param {String} option.date --date | date of report on ezunpaywall
+ * @param {Boolean} option.list -L --list | list of report on ezunpaywall
+ */
+const updateReport = async (option) => {
+  let report;
+
+  if (option.list) {
+    const reports = await getReports();
+    if (!reports.length) {
+      logger.info('No reports on ezunpaywall');
+      process.exit(0);
+    }
+    let filename;
+    filename = await inquirer.prompt([{
+      type: 'list',
+      pageSize: 5,
+      name: 'files',
+      choices: reports,
+      message: 'files',
+      default: reports.slice(),
+      source: (answersSoFar, input) => new Promise((resolve) => {
+        const result = reports.files
+          .filter((file) => file.toLowerCase().includes(input.toLowerCase()));
+        resolve(result);
+      }),
+    }]);
+    filename = filename.files;
+
+    report = await getReport(filename, {});
+    console.log(JSON.stringify(report, null, 2));
+    process.exit(0);
+  }
+
+  if (option.file) {
+    report = await getReport(option.file, {});
+    console.log(JSON.stringify(report, null, 2));
+    process.exit(0);
+  }
+
+  if (option.latest) {
+    report = await getReport('', { latest: true });
+    console.log(JSON.stringify(report, null, 2));
+    process.exit(0);
+  }
+  process.exit(0);
+};
+
+/**
+ * get status of update process
+ * @param {Boolean} option.verbose --verbose | show loading bar
+ *
+ */
+const updateStatus = async (option) => {
+  const ezunpaywall = await connection();
+  let res;
+  try {
+    res = await ezunpaywall({
+      method: 'GET',
+      url: '/api/update/status',
+    });
+  } catch (err) {
+    logger.errorRequest(err);
+    process.exit(1);
+  }
+
+  const status = res?.data?.inUpdate;
+  if (!status) {
+    logger.info('No update is in progress');
+    logger.info('Use ezu update report --latest to see the latest report');
+    process.exit(0);
+  } else {
+    logger.info('An update is being done');
+    if (option.verbose) {
+      await verbose();
+    } else {
+      const state = await getState('', true);
+      console.log(JSON.stringify(state, null, 2));
+      process.exit(0);
+    }
+  }
+  process.exit(0);
```

Comment:
Unnecessary `else` block

```suggestion
  }
  
  logger.info('An update is in progress');
  
  if (option.verbose) {
    await verbose();
  } else {
    const state = await getState('', true);
    console.log(JSON.stringify(state, null, 2));
  }
  process.exit(0);
```